stdlib/2and3/csv.pyi

```diff
@@ -22,25 +22,25 @@ _Dialect = Union[str, Dialect]
 _DictRow = Dict[str, Any]
 
 class Dialect(object):
-    delimiter = ...  # type: str
-    quotechar = ...  # type: Optional[str]
-    escapechar = ...  # type: Optional[str]
+    delimiter = ...  # type: Text
```

Comment:
This is a case where this breaks with `unicode_literals`.  I know `unicode_literals` has effectively been deprecated, but we still have a ton of code that makes use of it and we can't rip it out before upgrading to Python 3 without risking further breakage.

I suppose all of these fields could be modeled with as a `@property` who accepts a `Union[str, Text]` for the setter and return a `str`